vector/src/main/res/values/theme_status.xml

```diff
@@ -8,12 +8,18 @@
         <!-- default background color -->
         <item name="riot_primary_background_color">@color/riot_primary_background_color_status</item>
 
+        <!-- waiting view background -->
+        <item name="waiting_background_color">#AAAAAAAA</item>
+
```

Comment:
The comment `<!-- Only setting the items we need to override, otherwise inheriting -->` can be removed from this file. It comes from the `Black` theme, which inherit the `Dark` theme, and not directly an `AppCompat` theme.
Also, consider adding every missing elements form theme `Theme.Vector.Light` here, to be sure there will be no other crash in the future. Keeping the same order please :)